.github/workflows/test_install.yml

```diff
@@ -0,0 +1,51 @@
+on: [push]
+
+jobs:
+  installer_test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        verifier-release: [main, v0.0.1, v1.0.0, v1.0.1, v1.0.2, v1.1.0, v1.1.1, v1.2.0, v1.3.0]
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+      - name: Install verifier ${{ matrix.verifier-release }}
+        uses: ./actions/install
+        with:
+          verifier-release: ${{ matrix.verifier-release }}
+      - name: Run verifier help
+        run: slsa-verifier --help
+        shell: bash
+  invalid_version_test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+      - shell: bash
+        run: |
+          if ! $GITHUB_WORKSPACE/actions/install/installer.sh catdog ; then
+            exit 0
+          fi
+          exit 1
+  invalid_hash_test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+      - shell: bash
+        run: |
+          if ! $GITHUB_WORKSPACE/actions/install/installer.sh test-force-checksum-error ; then
```

Comment:
why not use `uses: ./actions/install` as well?

Reply:
I want the job to succeed iff the step fails, and afaict GitHub doesn't support that. I found an open issue for a feature that would enable this use case, but it doesn't look like GH is acting on it (https://github.com/actions/toolkit/issues/399).

Reply:
Maybe you could use a verification job even when this job fails. See some example in https://github.com/slsa-framework/example-package/blob/main/.github/workflows/e2e.generic.push.main.default.slsa3.yml#L92-L106